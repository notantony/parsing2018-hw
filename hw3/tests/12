f a b =
    while != 0 b
        (= a % a b
        = t a
        = a b
        = b t)