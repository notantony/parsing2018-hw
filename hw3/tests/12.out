void f(int a, int b) {
    int t;
    while ((0 != b))
    {
        a = (a % b);
        t = a;
        a = b;
        b = t;
    }
}
